assets/stylesheets/common/campaign.scss

```diff
@@ -301,6 +301,16 @@ html:not(.mobile-view) .subscriptions-campaign-topic-footer .campaign-banner {
   }
 }
 
+// Topic Footer Version + Sidebar visible
+html:not(.mobile-view)
+  body.has-sidebar-page
+  .subscriptions-campaign-topic-footer
+  .campaign-banner {
+  @media screen and (max-width: 1285px) {
```

Comment:
I thought we had a linting rule for the media queries to be outside the selector...

Reply:
Hmm, I haven’t seen any lint warnings about that rule, but I’m happy to wrap the selectors in the media query if that’s what you all prefer.